### ISSUE TITLE
[Bug]: Override of "Drawing.prototype._createText" does not exist

### ISSUE THREAD

**Issue by magroader** [arcanistzed/custom-fonts#21]
### Expected Behavior

No errors in console when loading with module enabled

### Current Behavior

Console error, module doesn't seem to work as a result

### Steps to Reproduce

1. Install Foundry 10.288
2. Enable custom-fonts module
3. Run a world
4. See console error in console window

### Context

I am not sure exactly what the override of _createText is trying to do, but I suspect it can probably be replaced using this:

```
  /**
   * Configure the default Token text style so that it may be reused and overridden by modules
   * @type {PIXI.TextStyle}
   */
  canvasTextStyle: new PIXI.TextStyle({
...
```

Commenting out the wrapping of _createText makes (most features of?) the module work.

I used "Find the Culprit" to verify this happens with no other modules (except libWrapper) enabled.

### Version

0.7.2

### Foundry VTT Version

10.288

### Operating System

Windows

### Browser / App

Other

### Game System

dnd5e

### Modules Disabled

- [X] I can confirm that disabling all of my other modules doesn't fix this issue.

**Comment by arcanistzed** [comment]
Yes, the module is meant to be used on V10 with the Sharper Text Drawings feature disabled in settings. I have been asking for clarifications on why this was changed, and I am still waiting for a response. This feature was working when I originally updated the module for V10 compatibility during the testing phase right before their stable release...

**Comment by arcanistzed** [comment]
Will be resolved after foundryvtt/foundryvtt#8502 is released

**Comment by magroader** [comment]
New update just dropped, and it needs some slight tweaks.

This seems to work for me. It no longer sets `this.text`, which I'm not sure if that was required, but I suspect it'll need to go elsewhere.

```
		libWrapper.register(
			CustomFonts.ID,
			"Drawing.prototype._getTextStyle",
			function (wrapped) {
				return new PIXI.TextStyle(
					mergeObject(wrapped(), {
						dropShadow: false,
						strokeThickness: 0,
						padding: 100,
					})
				);				
			},
			"WRAPPER"
		);
```

**Comment by arcanistzed** [comment]
Yep, that looks perfect. No need for the `this.text` stuff. I wonder if there is a way to configure the `text.resolution` as well.
